crates/bevy_ecs/src/schedule/state.rs

```diff
@@ -230,6 +231,31 @@ impl<T: Component + Clone + Eq> State<T> {
         Ok(())
     }
 
+    /// Schedule a state change that replaces the full stack with the given state.
```

Comment:
This comment is the exact same as `set_next`, which makes me think it is incorrect.

Reply:
I think the old comment needs to be updated to reflect the new `next` behavior.